packages/babel-core/src/config/files/configuration.ts

```diff
@@ -90,8 +90,7 @@ function* readConfigCode(
 
   // @ts-expect-error todo(flow->ts)
   if (typeof options.then === "function") {
-    // @ts-expect-error We use ?. in case options is a thenable
-    // but not a promise
+    // @ts-expect-error We use ?. in case options is a thenable but not a promise
```

Comment:
This is just to trigger a babel/core release, as per https://github.com/babel/babel/issues/16694#issuecomment-2256979076

Reply:
Lol, I have a dedicate PR for that purpose: https://github.com/babel/babel/pull/16696